suites/reef/rgw/tier-2_rgw_ms_archive_with_haproxy.yaml

```diff
@@ -665,3 +665,16 @@ tests:
       polarion-id: CEPH-83574621
       module: sanity_rgw_multisite.py
       name: Change bucket ownership to a different user when encryption is enabled
+
+  - test:
```

Comment:
We need this testcase to be run as a part of other builds of ceph..

Reply:
I mean don't we need this on quincy or squid?

Reply:
I have added as part of squid also , pls review